package.json

```diff
@@ -24,6 +24,7 @@
     "@heroicons/react": "^2.1.5",
     "@mdx-js/react": "^3.0.0",
     "clsx": "^2.0.0",
+    "docusaurus": "^1.14.7",
```

Comment:
delete line number `27` it's not part of our `package.json`

Reply:
@ajay-dhangar Should I add package.json to gitignore ?